ios/browser/api/features/BUILD.gn

```diff
@@ -19,6 +19,7 @@ source_set("features") {
     "//brave/components/brave_search_conversion:brave_search_conversion",
     "//brave/components/brave_shields/core/common:common",
     "//brave/components/brave_sync:features",
+    "//brave/components/brave_user_agent/common:common",
```

Comment:
Not sure why we're explicitly appending `:common` here on other entries but we dont need that